src/modules/Lan.rb

```diff
@@ -265,6 +265,10 @@ def Read(cache)
         return true
       end
 
+      system_config = Y2Network::Config.from(:sysconfig)
+      add_config(:system, system_config)
+      add_config(:yast, system_config.copy)
```

Comment:
can we think about a COW approach? As Config object will grow this will slow module loading as it already happened in Hosts module (and lead to system crash or unusable system with /etc/hosts with > 1000 lines) which used similar approach.

Reply:
I worry that if we want to do COW, we need something like https://evilmartians.com/chronicles/clowne-clone-ruby-models-with-a-smile as I expect there a lot of exceptions what can be dupped and what not.